### PR TITLE
add Console commands for ChainGovernance Precompiled Contract

### PR DESCRIPTION
**Pull request by sulenn**

### CODE REVIEW

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- cmd/commandline/chain_governance.go  1
         

Clones added
============
- cmd/commandline/chain_governance.go  11
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5891563&bid=19289296)

File: cmd/commandline/chain_governance.go [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+	"fmt"
+	"strconv"
+
+	"github.com/FISCO-BCOS/go-sdk/precompiled/chaingovernance"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/spf13/cobra"
+)
+
+// =========== chain governance ==========
```
Comment (bxq2011hust) [inline cmd/commandline/chain_governance.go:13]:
```suggestion

```

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- cmd/commandline/chain_governance.go  1
         

Clones added
============
- cmd/commandline/chain_governance.go  11
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5891563&bid=19289296)

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- cmd/commandline/chain_governance.go  1
         

Clones added
============
- cmd/commandline/chain_governance.go  11
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5891563&bid=19289296)

**Pull request merged**